scripts/deploy.sh

```diff
@@ -122,6 +123,15 @@ if [[ ! -f /etc/openstack_deploy/user_osa_secrets.yml ]] && [[ -f /etc/openstack
   mv /etc/openstack_deploy/user_secrets.yml /etc/openstack_deploy/user_osa_secrets.yml
 fi
 
+if [[ "$DEPLOY_MAGNUM" == "yes" ]]; then
+  cat >> /etc/openstack_deploy/user_osa_secrets.yml <<'EOF'
```

Comment:
@chrishultin could you move this into a seperate file in tree. You can place it here:
https://github.com/rcbops/rpc-openstack/tree/mitaka-13.1/rpcd/etc/openstack_deploy
This script will move the file to /etc/openstack_deploy automatically. All that is left to do is to use pw-token-gen.py in here to populate the values.